cmd/juju/action/listoperations.go

```diff
@@ -265,7 +256,14 @@ func (s byId) Swap(i, j int) {
 	s[i], s[j] = s[j], s[i]
 }
 func (s byId) Less(i, j int) bool {
-	return s[i].OperationTag < s[j].OperationTag
+	// We expect IDs to be ints but legacy
+	// action still use UUIDs.
+	id1, err1 := strconv.Atoi(s[i].ID)
```

Comment:
Can we make this smarter to separate/group old UUID actions and new monotonic actions.

If _i_ is a UUID and _j_ is a monotonic-id then _i_ is less than _j_.
If _j_ is a UUID and _i_ is a monotonic-id then _j_ is less than _i_.

Reply:
UUID actions were last created in 2.7 and are likely pruned by now, but easy enough to add some extra logic